uiServices/xmlWriter.go

```diff
@@ -0,0 +1,37 @@
+package services
+
+import (
+	"encoding/xml"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/xtracdev/automated-perf-test/perfTestUtils"
+	"os"
+	"fmt"
+)
+
+func writerXml(config perfTestUtils.Config, configPathDir string) bool {
+	filename := fmt.Sprintf("%s%s.xml", configPathDir, config.APIName)
+	//validate that directory exists
+	_, err := os.Stat(configPathDir)
```

Comment:
you have a public function that already does this. Also consider that you have already checked for this and could remove the check from this function.